### PR TITLE
Mo tiny fixes

### PR DESCRIPTION
**Pull request by stelmo**
This PR implements some small changes

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit c6c37d7c is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 13d93a23 is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 0ef191e0 is formatted properly.

File: src/analysis/modifications/generic.jl [hunk 1]
```diff
@@ -25,7 +25,7 @@ change_constraint(id::String, lb, ub) =
     end
 
 """
-    change_objective(new_objective::Union{String,Vector{String}}; weights=[], sense=MOI.MAX_SENSE)
+    change_objective(new_objective::Union{String,Vector{String}}; weights=[], sense=COBREXA.MOI.MAX_SENSE)
```
Comment (exaexa) [inline src/analysis/modifications/generic.jl:28]: we've got our aliases for these, `src/base/constants.jl` names them as MAX_SENSE and MIN_SENSE without any prefix business

File: src/base/types/FluxSummary.jl [hunk 2]
```diff
@@ -49,7 +49,7 @@ Export:
 ```
 """
 function flux_summary(
-    flux_result::Dict{String,Float64};
+    flux_result::Union{Nothing,Dict{String,Float64}};
```
Comment (exaexa) [inline src/base/types/FluxSummary.jl:52]: Maybe

File: src/base/types/FluxSummary.jl [hunk 3]
```diff
@@ -58,6 +58,13 @@ function flux_summary(
     large_flux_bound = _constants.default_reaction_bound,
     keep_unbounded = false,
 )
+    isnothing(flux_result) && return FluxSummary(
+        OrderedDict{String,Float64}(),
+        OrderedDict{String,Float64}(),
+        OrderedDict{String,Float64}(),
+        OrderedDict{String,Float64}(),
+    )
+
```
Comment (exaexa) [inline src/base/types/FluxSummary.jl:67]: perhaps better to have these as defaults in FluxSummary constructor?

Comment (exaexa) [reply src/base/types/FluxSummary.jl:67]: Other way: return `nothing :: Maybe{FluxSummary}`

File: src/base/types/FluxVariabilitySummary.jl [hunk 4]
```diff
@@ -50,7 +50,10 @@ function flux_variability_summary(
     biomass_strings = _constants.biomass_strings,
     exclude_biomass = false,
 )
-
+    isnothing(flux_result) && return FluxVariabilitySummary(
+        Dict{String,Vector{Union{Float64,Nothing}}}(),
+        Dict{String,Vector{Union{Float64,Nothing}}}(),
+    )
```
Comment (exaexa) [inline src/base/types/FluxVariabilitySummary.jl:56]: same here, if an empty FluxVarSummary makes sense, better have defaults.

**Review by exaexa — CHANGES_REQUESTED** [review]
minor tunings

File: src/analysis/modifications/generic.jl (hunk 1 above)
Comment (stelmo) [reply src/analysis/modifications/generic.jl:28]: fixed

Comment (exaexa) [reply src/analysis/modifications/generic.jl:28]: without hte COBREXA prefix actually. We removed lots of that in the other PR :D

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 9c86dfe5 is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 1cf394f7 is formatted properly.

**Review by exaexa — APPROVED** [review]

**Pull request merged**